[Codemod][Codemod deprecated unittest asserts] fbcode//caffe2/test

cricalix opened the pull request:
Summary:
In Python 3.2, a number of asserts were deprecated.

In Python 3.11, these asserts are deleted completely. The files in this change still use the deprecated asserts.

Switch over to the supported syntax for 3.2 onwards.

Test Plan: Tested on the internal test suite runner.

Differential Revision: D33503694

albanD reviewed (approved):
LGTM
Sorry for the delay!

cricalix committed:
[Codemod][Codemod deprecated unittest asserts] fbcode//caffe2/test (#71708)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/71708

In Python 3.2, a number of asserts were deprecated.

In Python 3.11, these asserts are deleted completely. The files in this change still use the deprecated asserts.

Switch over to the supported syntax for 3.2 onwards.

Test Plan: Tested on the internal test suite runner.

Reviewed By: ajtulloch

Differential Revision: D33503694

fbshipit-source-id: fc81d4019146f42f1a792280b78ffece6b88de7b
